diff --git a/rootdir/init.rc b/rootdir/init.rc
index d78f77a..18e8bd5 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -500,7 +500,7 @@ service ril-daemon /system/bin/rild
     socket rild stream 660 root radio
     socket rild-debug stream 660 radio system
     user root
-    group radio cache inet misc audio sdcard_rw qcom_oncrpc qcom_diag log
+    group radio cache inet misc audio sdcard_r sdcard_rw qcom_oncrpc qcom_diag vpn net_admin log
 
 service surfaceflinger /system/bin/surfaceflinger
     class main
